source/stylesheets/refills/_accordion-tabs.scss

```diff
@@ -1,30 +1,21 @@
 .accordion-tabs {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-background-color: white !default;
-  $base-spacing: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  $dark-gray: #333 !default;
-  $light-gray: #DDD !default;
-  $medium-screen: em(640) !default;
-  //////////////////////////////////////////////////////////////////////////////////
-
-  $tab-border: 1px solid $base-border-color;
-  $tab-content-background: lighten($light-gray, 10);
-  $tab-active-background: $tab-content-background;
-  $tab-inactive-color: $base-background-color;
-  $tab-inactive-hover-color: darken($light-gray, 5);
-  $tab-mode: $medium-screen;
+  $tab-spacing:            if(global-variable-exists(base-spacing), $base-spacing, 1.5em);
+  $tab-accent-color:       if(global-variable-exists(action-color), $action-color, #477dca);
+  $tab-border:             if(global-variable-exists(base-border), $base-border, 1px solid gainsboro);
```

Comment:
Not sure if the search thing is a good enough argument though. I search for stuff by highlighting it then hitting a shortcut. Those who type would probably rather type `#dcdcdc` than have to remember how to spell that color name. Not a big deal though.